test/e2e/versionskew/operator.go

```diff
@@ -202,10 +216,18 @@ var _ = deploy.DescribeForSome("versionskew", func(d *deploy.Deployment) bool {
 	}
 
 	It("downgrade [Slow]", func() {
+		if d.HasOLM {
+			Skip("OLM yet not support operator downgrade!")
```

Comment:
"OLM does not support operator downgrade yet."

Reply:
Done.